docs/SetupDatabaseForAzure.md

```diff
@@ -1,72 +1,176 @@
 
-* To create an Azure Database for MySQL, 
-  * For Single Server deployment, follow this [Azure official guide](https://docs.microsoft.com/en-us/azure/mysql/quickstart-create-mysql-server-database-using-azure-portal).
-  * For creating a flexible server deployment, follow this [Azure official guide](https://docs.microsoft.com/en-us/azure/mysql/flexible-server/quickstart-create-server-portal).
-* (Optional) Configure advanced monitoring services with the [Azure official guide](https://docs.microsoft.com/en-us/azure/mysql/concepts-monitoring), monitoring is enabled in Azure Database for MySQL by default.
-* (Optional) To scale resources like vCores and storage after the deployment,
-  * Follow the [Azure official guide](https://docs.microsoft.com/en-gb/azure/mysql/concepts-pricing-tiers#scale-resources) for learning more about scaling single server resources.
-  * Follow the [Azure official guide](https://docs.microsoft.com/en-gb/azure/mysql/flexible-server/concepts-compute-storage#scale-resources) for learning more about scaling flexible server resources.
-  
-Note:-
+If you want to change some backup configurations like the backup retention period, you can configure it. Please refer to the official document for more details.
 
-* The [Storage auto-grow](https://docs.microsoft.com/en-gb/azure/mysql/concepts-pricing-tiers#storage-auto-grow) feature automatically increases storage without impacting the workload and is enabled by default in server configuration while creating an Azure Database for MySQL instance.
+* [Backup and restore in Azure Database for MySQL Flexible Server](https://learn.microsoft.com/en-us/azure/mysql/flexible-server/concepts-backup-restore)
 
+#### Configure monitoring (Recommended in the production environment)
+
+You can configure the monitoring of Azure Database for MySQL using its native feature. Please refer to the official document for more details.
+
+* [Monitor Azure Database for MySQL Flexible Server](https://learn.microsoft.com/en-us/azure/mysql/flexible-server/concepts-monitoring)
+
+#### Disable public access (Recommended in the production environment)
+
+You can configure **Private access (VNet Integration)** as a **Connectivity method**. Please refer to the official document for more details.
+
+* [Connectivity and networking concepts for Azure Database for MySQL - Flexible Server](https://learn.microsoft.com/en-us/azure/mysql/flexible-server/concepts-networking)
+
+You can access the database server from the Scalar product pods on your AKS cluster as follows.
+
+* Create the database server on the same VNet as your AKS cluster.
+* Connect the VNet for the database server and the VNet for the AKS cluster for the Scalar product deployment using [Virtual network peering](https://learn.microsoft.com/en-us/azure/virtual-network/virtual-network-peering-overview). (// TODO: We need to test this feature with Scalar products.)
```

Comment:
We have not yet tested this feature with Scalar products.
I will do the test and update the document as another task.